.github/workflows/driver_ci.yaml

```
@@ -0,0 +1,135 @@
+name: Driver Build
+on:
+  push:
+    branches:
+      - develop
+      - master
+  pull_request:
+    branches:
+      - develop
+      - master
+jobs:
+  prepare_env:
+   name: "Prepare env"
+   runs-on: ubuntu-latest
+   steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.9.7'
+      - uses: FranzDiebold/github-env-vars-action@v2
+      - name: Setup dependencies
+        id: setup_dependencies
+        run: |
+          scripts/ci/github_actions/driver/setup_ci_dependencies.sh
+        env:
+          build_number: "${{ github.run_number }}"
+      - uses: actions/cache@v2
+        with:
+          path: ${{ env.pythonLocation }}
+          key: ${{ env.pythonLocation }}-${{ hashFiles('dev-requirements.txt') }}
+      - name: Save dependencies files
+        uses: actions/upload-artifact@v2
+        with:
+          name: ci-dependencies
+          path: |
+            /home/runner/.bash_profile
+          retention-days: 1
+   outputs:
+     docker_image_branch_tag: "${{ steps.setup_dependencies.outputs.docker_image_branch_tag }}"
+     driver_images_specific_tag: "${{ steps.setup_dependencies.outputs.driver_images_specific_tag }}"
+
+  csi_controller_static_code_analysis:
+    runs-on: ubuntu-latest
+    name: "CSI controller static code analysis"
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: CSI-controller- static code analysis
```

```suggestion
      - name: CSI-controller: static code analysis
```
could you align the names to the [ones in the jenkinsfile](https://github.com/IBM/ibm-block-csi-driver/blob/a05503f9c8f49363ea1e32c16f870f055c2bc857/scripts/ci/Jenkinsfile#L17), as much as possible?
unless you know of an easy way to maintain them in one place

---

this would also mean:
`k8s yamls validation` should be:
`K8s yamls validation` (in both files)